k8s/backend-deployment.yml

```
@@ -10,25 +10,23 @@ spec:
+          env:
+            - name: NODE_ENV
+              value: production
+            - name: MONGODB_URI
+              value: "mongodb://mongoadmin:mongoadmin@mongodb:27017/dbname?authSource=admin"
+            - name: JWT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: chatapp-secrets
+                  key: jwt
+            - name: PORT
+              value: "5001"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)